tests/lean/run/PPTopDownAnalyze.lean

```diff
@@ -310,7 +310,8 @@ def takesStrictMotive ⦃motive : Nat → Type⦄ {n : Nat} (x : motive n) : mot
 def arrayMkInjEqSnippet :=
   fun {α : Type} (xs : List α) => Eq.ndrec (motive := fun _ => (Array.mk xs = Array.mk xs)) (Eq.refl (Array.mk xs)) (rfl : xs = xs)
 
-#testDelabN arrayMkInjEqSnippet
+-- TODO: fix following test
+-- #testDelabN arrayMkInjEqSnippet
```

Comment:
@Kha This issue was being masked by `abstractNestedProofs`. It was incorrectly abstracting the `arrayMkInjEqSnippet` since it was marked as `def`. So, it was not testing anything since the body was just an application after the abstraction step.

Reply:
There is no rush to fix this one.

Reply:
Good, because I don't know anything about `pp.analyze` :)